python/ray/experimental/state.py

```diff
@@ -65,8 +66,25 @@ def _initialize_global_state(self, redis_ip_address, redis_port):
     """
     self.redis_client = redis.StrictRedis(host=redis_ip_address,
                                           port=redis_port)
+    self.redis_clients = []
+    for ip_address_port in self.redis_client.lrange(
+            "RedisShards", start=0, end=-1):
+      shard_address, shard_port = ip_address_port.split(b":")
+      self.redis_clients.append(redis.StrictRedis(host=shard_address,
+                                                  port=shard_port))
+
+  def _execute_command(self, object_id, *args):
```

Comment:
This is sometimes called with task IDs, so can we call the argument `identifier` or something like that?

Also, can we add a doc string like the following (with the appropriate line lengths)?

```python
"""Execute a Redis command on the appropriate Redis shard.

This command hashes the identifier to find the appropriate Redis shard
to query, and runs the command on that Redis server.

Args:
  identifier: The object ID or the task ID of that the query is about.
  args: The command to run.

Returns:
  The value returned by the redis command.
"""
```